manifests/istio-control/istio-discovery/templates/deployment.yaml

```diff
@@ -98,11 +98,6 @@ spec:
             initialDelaySeconds: 5
             periodSeconds: 5
             timeoutSeconds: 5
-          envFrom:
```

Comment:
does this work currently? I thought we were really striving to remove environment variables from the system. I think simplifying by removing unnecessary configmaps makes a ton of sense.